website/docs/manual/integrate/04-flutter-package.md

```diff
@@ -0,0 +1,5 @@
+# Flutter package
+
+## Quickstart
+
+To have a working quickstart run use the cli tool and run `flutter_rust_bridge_codegen create flutter_package --template plugin` or alternatively copy https://github.com/fzyzcjy/flutter_rust_bridge/tree/master/frb_example/flutter_package
```

Comment:
nit: maybe 

- [ ] reverting doc changes except for the change in cargokit.md
- [ ] In `quickstart.md` the "flutter package" tab, mimic the "add to existing" or "craete new" tab to explain the one-line command